src/main.rs

```diff
@@ -354,6 +359,10 @@ async fn main() {
                 }
             }
         }
+        Commands::Config => {
+            let conf = LibraryConfig::get_config_path().to_string_lossy().to_string();
```

Comment:
Seems arbitrary to use the library config for this. We should use the config path constant/flag for this

Reply:
The point is that it'll print when using the default as well.

Reply:
Then there should be a clear-cut way to get the default/custom config path, I don't like arbitrarily using a single config since it could change in the future